packages/marshal/src/rankOrder.js

```diff
@@ -46,9 +46,46 @@ const { entries, fromEntries, setPrototypeOf, is } = Object;
  */
 const sameValueZero = (x, y) => x === y || is(x, y);
 
+/**
+ * @param {any} left
+ * @param {any} right
+ * @returns {RankComparison}
+ */
 export const trivialComparator = (left, right) =>
```

Comment:
This could be simply called `compare` since it is the default JavaScript comparator made flesh.

Reply:
Already exported from package before this PR, so I'd rather not introduce a compat problem for a minor improvement to this name. That said, were it not for the (also minor) compat pain, I do like `compare` better.

Reply:
Resolving.